security/hosts_multitenancy.adoc

```diff
@@ -0,0 +1,105 @@
+and its dependencies as a unit. Containers also make it easy to deploy multiple
+applications on a single host, using the kernel and the docker runtime to spin
+up each individual container.
+
+To take full advantage of this packaging and deployment technology, an
+operations team needs the right environment for running containers. Operations
+needs an operating system (OS) that can secure containers at the boundaries:
+securing the host kernel from container escapes and securing containers from
+each other.
+
+Because {product-title} runs on Red Hat Enterprise Linux (RHEL) and RHEL Atomic
+Host, the following concepts apply by default to any deployed {product-title}
+cluster and are at the core of what make containers secure on the platform.
+
+[[security-hosts-how-containers-are-secured-on-rhel]]
+== How Containers are Secured on RHEL
+
+In Linux, containers are really just a special type of process, so your approach
+to securing containers should be the same as your approach to securing any
+running process. Dropping privileges is important and still best practice.
+Containers should run as a non-root user. There are multiple levels of security
+and a lot of history involved in securing containers on Linux.
+
+Linux namespaces, SELinux, CGroups and Secure Computing Mode (seccomp) are four
+of the security features available for securing containers running on RHEL and
+RHEL Atomic Host:
+
+- _Linux namespaces_ provide the fundamentals of container isolation. A namespace
+makes it appear to the processes within the namespace that they have their own
+instance of global resources. Namespaces provide the abstraction that gives the
+impression you are running on your own operating system when you are inside a
+container.
+
+- _SELinux_ provides an additional layer of security to keep containers isolated
+from each other and from the host. SELinux allows administrators to enforce
+mandatory access controls (MAC) for every user, application, process, and file.
+SELinux is the brick wall that is going to stop you if you manage to break out
+of (accidentally or on purpose) the namespace abstraction.
+
+- _CGroups_ (or control groups) limit, account for, and isolate the resource usage
+(CPU, memory, disk I/O, network, etc.) of a collection of processes. CGroups are
+used to ensure your container will not be stomped on by another container on the
+same host.
+
+- Finally, a _secure computing mode (seccomp)_ profile can be associated with a
+container to restrict available system calls.
+
+You can further enhance security for your applications and infrastructure by
+deploying your containers to RHEL Atomic Host. RHEL Atomic Host reduces the attack
+surface by minimizing the host environment and tuning it for containers.
+
+As a proof point of the security features available to you with RHEL and RHEL
+Atomic Host, RHEL 7.1 recently received Common Criteria certification, including
+certification of the Linux Container Framework Support.
```

Comment:
It's almost like the above should have to exclamation points...

I'd suggest this be changed to something like "RHEL 7.1 also comes with X security options." and if you're still keen on keeping this para in here, I'd include a link to the features that got RHEL this certification.